www/front_src/src/Resources/Graph/Performance/Graph/AnchorPoints/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import { ScaleLinear, ScaleTime } from 'd3-scale';
+import { isEmpty, isNil, or, prop } from 'ramda';
+
+import { Line, TimeValue } from '../../models';
+import { getUnits, getYScale } from '../../timeSeries';
+
+import AnchorPoint from './Point';
+
+interface Props {
+  leftScale: ScaleLinear<number, number>;
+  lines: Array<Line>;
+  rightScale: ScaleLinear<number, number>;
+  timeValue: TimeValue;
+  xScale: ScaleTime<number, number>;
+}
+
+const AnchorPoints = ({
+  rightScale,
+  leftScale,
+  xScale,
+  lines,
+  timeValue,
+}: Props): JSX.Element => {
+  const [, secondUnit, thirdUnit] = getUnits(lines);
```

Comment:
what is the meaning of a blank comma here?

Reply:
Here we don't care about the first element, thus, we don't give a name to it to avoid ESlint yelling about `no unused variable` while destructuring an array